<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>客户端</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>文件目录</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QTreeView" name="treeView"/>
              </item>
             </layout>
            </widget>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>列表显示</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLineEdit" name="pathLineEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QListView" name="listView"/>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>详细信息</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QTextEdit" name="datailTextEdit"/>
              </item>
             </layout>
            </widget>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>控制台</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QTextBrowser" name="textBrowser"/>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>服务器</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actConnect"/>
   <addaction name="actDisconnect"/>
   <addaction name="actDownload"/>
  </widget>
  <action name="actConnect">
   <property name="text">
    <string>连接服务器</string>
   </property>
   <property name="toolTip">
    <string>连接服务器</string>
   </property>
  </action>
  <action name="actDisconnect">
   <property name="text">
    <string>断开连接</string>
   </property>
   <property name="toolTip">
    <string>断开连接</string>
   </property>
  </action>
  <action name="actUpload">
   <property name="text">
    <string>上传文件</string>
   </property>
  </action>
  <action name="actDownload">
   <property name="text">
    <string>下载文件</string>
   </property>
   <property name="toolTip">
    <string>下载文件</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
